suite-native/ethereum-tokens/src/ethereumTokensSelectors.ts

```diff
@@ -31,3 +40,30 @@ export const selectIsEthereumAccountWithTokensWithBalance = (
         A.isNotEmpty(account.tokens.filter(filterTokenHasBalance))
     );
 };
+
+export const selectEthereumAccountToken = (
+    state: AccountsRootState,
+    accountKey: AccountKey,
+    tokenSymbol?: EthereumTokenSymbol,
+): TokenInfo | null => {
+    const account = selectAccountByKey(state, accountKey);
+    return account?.tokens
```

Comment:
nit: `if (G.isNullable(account.tokens)) return null`

Reply:
fixed in 4d5c05278